src/explore-education-statistics-admin/src/pages/release/data/components/ReorderList.tsx

```diff
@@ -0,0 +1,209 @@
+import styles from '@admin/pages/release/data/components/ReorderList.module.scss';
```

Comment:
Nice Component to have!

Not knowing much about how the other drag n' drop components work that we currently have, is this potentially something that we can adapt to reuse in the future for other purposes?  At the moment I guess it's tied in around Filters and Indicators explicitly.

Reply:
Potentially... though I had enough problems getting typescript to let me make it generic enough to work with both indicators and filters so might be more trouble than it's worth!